nix/pkgs/haskell/haskell.nix

```diff
@@ -182,7 +183,7 @@ let
 
           # See https://github.com/input-output-hk/iohk-nix/pull/488
           cardano-crypto-praos.components.library.pkgconfig = lib.mkForce [ [ libsodium-vrf ] ];
-          cardano-crypto-class.components.library.pkgconfig = lib.mkForce [ [ libsodium-vrf ] ];
+          cardano-crypto-class.components.library.pkgconfig = lib.mkForce [ [ libsodium-vrf secp256k1 ] ];
```

Comment:
nix/pkgs/haskell/haskell.nix:
        186: Line too long (106 instead of 100)

1 errors found
builder for '/nix/store/5h547fappwcwkd32xckp8ga6kzp46ynb-editorconfig-checker.drv' failed with exit code 1

Reply:
So I guess I can just break the line?

Reply:
Yes i think so